libkmod/libkmod.c

```diff
@@ -179,6 +180,9 @@ static char *get_kernel_release(const char *dirname)
 	if (asprintf(&p, "%s/%s", dirname_default_prefix, u.release) < 0)
 		return NULL;
 
+	if ((is_dir(p) <= 0) && asprintf(&p, "%s/%s", dirname_fallback_prefix, u.release) < 0)
```

Comment:
Regardless of being accepted or not: This leaks memory in `p` from previous `asprintf` call.